fhirstore/fhirstore.py

```diff
@@ -278,134 +280,75 @@ def search(self, search_args: SearchArguments):
             - params: search parameters as returned by the API. For a simple
             search, the parameters should be of the type {"key": "value"}
             eg: {"gender":"female"}, with possible modifiers {"address.city:exact":"Paris"}.
-            If a search is made one field with multiple arguments (eg: language is French 
-            OR English), params should be a payload of type {"multiple": {"language": 
-            ["French", "English"]}}. 
-            If a search has more than one field queried, params should be a payload of 
+            If a search is made one field with multiple arguments (eg: language is French
+            OR English), params should be a payload of type {"multiple": {"language":
+            ["French", "English"]}}.
+            If a search has more than one field queried, params should be a payload of
             the form: {"address.city": ["Paris"], "multiple":
             {"language": ["French", "English"]}}.
         Returns: A bundle with the results of the search, as required by FHIR
         search standard.
         """
-        self.validate_resource_type(search_args.resource_type)
-
-        core_query = build_core_query(search_args.core_args)
-        bundle = Bundle()
+        if resource_type:
+            self.validate_resource_type(resource_type)
 
-        if search_args.formatting_args["is_summary_count"]:
-            hits = self.es.count(
-                body={"query": core_query}, index=f"fhirstore.{search_args.resource_type.lower()}",
+        search_context = SearchContext(self.search_engine, resource_type)
+        fhir_search = Search(search_context, query_string=query_string, params=params)
+        try:
+            return fhir_search()
+        except elasticsearch.exceptions.NotFoundError as e:
+            return OperationOutcome(
+                **{
```

Comment:
The `**` can be confusing

Maybe 
```
issue = {...}
OperationOutcome(issue=[issue])
```
would be clearer

Reply:
much better indeed! thanks